File: java/src/com/infoback/Alarm.java
package com.infoback;

import java.time.LocalDateTime;

public class Alarm
{
  private LocalDateTime alarmTime;

  private String alarmName;

  public Alarm(LocalDateTime alarmTime, String alarmName)
  {
    this.alarmTime = alarmTime;
    this.alarmName = alarmName;
  }

  @Override
  public String toString()
  {
    return String.format("'%s' set for %s", this.alarmName, this.alarmTime);
  }

  public LocalDateTime getAlarmTime() {
    return alarmTime;
  }

  public void setAlarmTime(LocalDateTime alarmTime) {
    this.alarmTime = alarmTime;
  }

  public String getAlarmName() {
    return alarmName;
  }

  public void setAlarmName(String alarmName) {
    this.alarmName = alarmName;
  }

}

File: dotnet/net5/Watches-V5/Alarm.cs
using System;

namespace WatchesV5
{

    public class Alarm
    {
        public DateTime AlarmTime { get; set; }
        public string AlarmName { get; set; }

        public Alarm(DateTime alarmTime, string alarmName)
        {
            AlarmTime = alarmTime;
            AlarmName = alarmName;
        }

        public override string ToString()
        {
            return $"'{AlarmName}' set for {AlarmTime}";
        }
    }
}